target/linux/ramips/image/mt7620.mk

```diff
@@ -257,6 +239,7 @@ TARGET_DEVICES += dlink_dwr-921-c1
 
 define Device/dlink_dwr-921-c3
   $(Device/dlink_dwr-921-c1)
+  DTS := mt7620_dlink_dwr-921-c1
```

Comment:
In parent Makefile DTS is set up with `DTS = ...`
Will I need `=` or `:=` here to overwrite that?

Reply:
Both of them should work. The difference between = and := is that whether the expression after it is expanded before or after the variable is used. This is just a plain string and there is nothing to expand anyway.